commands/developer-utils/GitHub-PRs-Inline.rb

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open Pull Requests
+# @raycast.mode inline
+# @raycast.refreshTime 5m
+# @raycast.packageName GitHub
+#
+# Optional parameters:
+# @raycast.iconDark images/github-logo-iconDark.png
+# @raycast.icon images/github-logo.png
+# @raycast.needsConfirmation false
+
+require 'json'
+require 'net/http'
+require 'uri'
+
+API_TOKEN = ''
+RED = 31
+GREEN = 32
+YELLOW = 33
+
+
+if API_TOKEN.empty?
+  puts 'No API token provided'
+  exit(1)
+end
+
+uri = URI('https://api.github.com/graphql')
+req = Net::HTTP::Post.new(uri)
+req['Authorization'] = "token #{API_TOKEN}"
+req.body = '{ "query": "query { viewer { pullRequests(first: 10, states:OPEN) { nodes { title number url } }}}" }'
```

Comment:
Keeping the spacing consistent :-)
```suggestion
req.body = '{ "query": "query { viewer { pullRequests(first: 10, states:OPEN) { nodes { title number url } } } }" }'
```